SU2_CFD/include/solvers/CSpeciesFlameletSolver.hpp

```diff
@@ -64,6 +64,46 @@ class CSpeciesFlameletSolver final : public CSpeciesSolver {
                                   CNumerics* visc_numerics, CConfig* config, unsigned short val_marker,
                                   bool cht_mode = false);
 
+  /*!
+   * \brief Reverse look-up to retrieve enthalpy value based on temperature and other controlling variables.
+   * \param[in] fluid_model - pointer to flamelet fluid model.
+   * \param[in] val_temp - temperature value used for reverse look-up.
+   * \param[in] val_enth - pointer to enthalpy value to be retrieved.
+   * \param[in] scalar_solution - local scalar solution.
+   * \param[out] Converged - 0 if Newton solver converged, 1 if not.
+   */
+  unsigned long GetEnthFromTemp(CFluidModel * fluid_model, su2double const val_temp, su2double * val_enth, const su2double * scalar_solution);
```

Comment:
const everything you can, see other functions too
```suggestion
  unsigned long GetEnthFromTemp(CFluidModel * fluid_model, su2double const val_temp, const su2double * val_enth, const su2double * scalar_solution);
```

Reply:
Ok, then the documentation should state that this variable is either "out" or "in,out", and it should be listed after all variables that are inputs.

Reply:
I updated the name of `val_enth` to be `val_enth_out` and set it as the last input variable for the `GetEnthFromTemp` function.